.github/workflows/snyk-security.yml

```
@@ -36,6 +36,12 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
+      - name: Set up Python environment and install dependencies
+        uses: actions/setup-python@v4
```

actionlint (1.7.7)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)